### ISSUE TITLE
Status Page - Events and UPTime Period

### ISSUE THREAD

**Issue by devzeroo** [louislam/uptime-kuma#1667]
### ⚠️ Please verify that this feature request has NOT been suggested before.

- [X] I checked and didn't find similar feature request

### 🏷️ Feature Request Type

Other

### 🔖 Feature description

Would be a great adition if Uptime Kuma had an option to show on Status Pages:

- Drill Down to a Monitor and show:

-- Overall UPtime per 24 Hr / 7Days / 30 Days / 90 Days.
-- Recent Events(Incidents) and Duration (Start time / End Time)

### ✔️ Solution

![screenshot-stats uptimerobot com-2022 05 20-10_16_24](https://user-images.githubusercontent.com/13603669/169535975-87daeeab-3a66-46b7-9754-23049c76be81.png)

![screenshot-stats uptimerobot com-2022 05 20-10_16_51](https://user-images.githubusercontent.com/13603669/169536071-75288292-be8a-461c-a4e7-f72fd7a2c149.png)

**Comment by owlrock7** [comment]
on the admin panel , i can see all online time for 24hours and 30 days. but at the status page ,  only latest 1 hour shown?

**Comment by Philabusterer** [comment]
Yes to drill down on Status Pages.  Since there is no way to create a "view only" user to see the all the details in the dashboard view, the status pages need to be able to show the same information as what is shown on the dashboard.

It should be possible to click on any monitor listed on a status page and see the same view that you can see in the dashboard view which shows events, response time, etc...  It should also display the same graph and have the same drop down options to show the graph view for various time periods.  Going back for 90 days or more also highly preferable to be able to see a longer historical view at a glance by looking at the graph.

**Comment by camilonova** [comment]
Would be lovely if when creating a status page you could select how many days (or timeframe) you want to show

**Comment by nathangemmill** [comment]
I would really like to be able to toggle the uptime period on the status page to a period of up to 1 year. 

It'd be great if you could set a default view on the status page of 24 hours but allow a dropdown to switch between the different views (like you can do in the dashboard).

**Comment by katzeprior** [comment]
I agree, current uptime is 100% because it shows the 24 hour only, the real 30 day uptime is 99.99% which would be better to show imo, and a 90 day and 1 year option would be cool as well even tho 1 year might be a bit to long timeframe.

**Comment by stianmandarin** [comment]
Has this issue been raised or discussed elsewhere? Is there a response to it? The current status page gives an entirely misleading statistic to those who view it, and as such, it loses most of it's value. It might as well just say either "up" or "down" instead giving a percentage since the current percentage basis is near meaningless.

**Comment by andreasbrett** [comment]
It's coming in 2.x https://github.com/louislam/uptime-kuma/pull/2750

**Comment by kokofixcomputers** [comment]
It would be really nice to have that.

**Comment by ryo9999** [comment]
I'd also like to be able to display the uptime for the entire period being measured.
It is easy to show the reliability of the server.
24hour/7days/30days/90days/1year/ALL
